### DISCUSSION TITLE
Development Environment Recommendations:  vscode and devcontainer?

### DISCUSSION THREAD

**Discussion by tleacmcsa** [openthread/openthread#10929] [Q&A] [answered]
What is the recommended development environment for openthread / ot-br-posix?  I was hoping there would be a .devcontainer and I could simply open the project in vscode and start banging away on my workstation :-D.

**Comment by jwhui** [comment, marked as answer]
We currently do not have a dev container for ot-br-posix. But we are open to contributions!
